frontend/packages/metal3-plugin/src/components/host-detail.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import { Row, Col } from 'patternfly-react';
+
+import { referenceForModel } from '@console/internal/module/k8s';
+import { DetailsPage } from '@console/internal/components/factory';
+import { navFactory } from '@console/internal/components/utils';
+import { units } from '@console/internal/components/utils/units';
+import { getName } from '@console/shared';
+
+import { BaremetalHostModel } from '../models';
+import MachineCell from './machine-cell';
+import {
+  getHostNICs,
+  isHostOnline,
+  getHostDescription,
+  getHostBMCAddress,
+  getHostCPUs,
+  getHostRAM,
+  getHostTotalStorageCapacity,
+} from '../selectors';
+
+type BaremetalHostDetailPageProps = {
+  namespace: string;
+  name: string;
+  match: any;
+};
+
+const BaremetalHostDetails = ({ obj }) => {
+  const nics = getHostNICs(obj);
+  const online = isHostOnline(obj);
+  const ips = nics.map((nic) => nic.ip).join(', ');
+
+  const statusClasses = classNames({
+    fa: true,
+    'co-icon-and-text__icon': online === true,
+    'fa-refresh': true,
+  });
+
+  const totalCapacity = units.humanize(
+    // The value from the selector is in GB
+    getHostTotalStorageCapacity(obj) * 1024 ** 3,
+    'decimalBytes',
+    true,
+  ).string;
+
+  return (
+    <div className="co-m-pane__body">
+      <h1 className="co-m-pane__heading">Baremetal Host Overview</h1>
+      <Row>
+        <Col lg={4} md={4} sm={4} xs={4} id="name-description-column">
```

Comment:
This is not the standard resource page layout. We should be consistent with other details pages.

Reply:
Here's a newer two-column mockup @honza:

![2-4-bmh-overview](https://user-images.githubusercontent.com/9122899/59703596-31699300-91c8-11e9-8077-de7d2b10cb45.png)

Reply:
Done